### DISCUSSION TITLE
updated user put route

### DISCUSSION THREAD

**Discussion by huilensolis** [CulinaryAlchemy/CulinaryAlchemy#132] [Announcements]
I have updated the user routing for the put method. These changes will be applied the next time I merge from "115-featbackend-fix-databse-relationships" to "main"

old route: 'user/id/<id>'
now route: 'user/<id>'

also working on a profile route!
@Jes015

**Comment by Jes015** [comment]
it's ok. add me as Reviewer for ```"115-featbackend-fix-databse-relationships" to "main"``` and i'll update it.
